Bigeye documentation (#842)
mozilla/data-docs: src/cookbooks/data_quality/issues_management.md

## Issues management

> The `Issues` tab filtering issues by parameters like severity or date, and reviewing details such as impacted metrics and tables. 

Reviewer: ```suggestion
The `Issues` tab allows filtering issues by parameters like severity or date, and reviewing details such as impacted metrics and tables. 
```

Users can also resolve, mute, or dismiss issues, helping maintain healthy data pipelines.

> <img src="../../assets/Bigeye/Issues-tab.png" style="max-width: 100%; height: 500px; width: 100%">

Reviewer: You shouldn't need the manual styling and can use markdown instead.
Please also include alt text:

```suggestion
![The Issues overview on BigEye, allowing to filter issues](../../assets/Bigeye/Issues-tab.png)
```

Same applies to the images below.

### View issue details

Once we click on the issue we can view a metric chart that displays a time series visualization of the alerting metric.

<img src="../../assets/Bigeye/Issues-2.png" style="max-width: 100%; height: 500px; width: 100%">

The metric chart in Bigeye displays a time series visualization of the alerting metric

<img src="../../assets/Bigeye/Issues-3.png" style="max-width: 100%; height: 500px; width: 100%">

### Status of Issue

Users can change the status to "Active," "Resolved," "Muted," or "Dismissed," depending on the issue's current state. This allows for better tracking and management of issues across data pipelines. Status updates are reflected in the timeline, providing a clear history of the issue's progression.

<img src="../../assets/Bigeye/Issues-4.png" style="max-width: 100%; height: 500px; width: 100%">

Use the Mute button above the timeline to mute an issue and stop being notified about subsequent issue alerts. The available mute options in the Bigeye are Unmute, 1 hour, 4 hours, 12 hours, 24 hours, or 1 week

<img src="../../assets/Bigeye/Issues-5.png" style="max-width: 100%; height: 500px; width: 100%">

> For more details, refer to the full documentation [`here`](https://docs.bigeye.com/docs/change-the-issue-status).

Reviewer: ```suggestion
For more details, refer to the [Bigeye documentation](https://docs.bigeye.com/docs/change-the-issue-status).
```
